lib/cursor.js

```diff
@@ -212,7 +236,7 @@ module.exports = {
       launder: function(s) {
         s = self.apos.launder.string(s);
 
-        if (!s.match(/^\d\d\d\d\-\d\d\-\d\d$/)) {
+        if (!s.match(/^\d\d\d\d-\d\d-\d\d$/)) {
           return null;
```

Comment:
Could this regex be a utility? Would avoid duplication.

Reply:
Oh, store it in a var? Sure.

Reply:
That would work. Or could be a fn like 

```
isMatch(s) {
    return s.match(/^\d\d\d\d-\d\d-\d\d$/);
}
```

I'd just name it closer to what your matching so that it's a little more explicit to what it matches.